shared/message/output/Buzzer.proto

```diff
@@ -0,0 +1,30 @@
+/*
+* This file is part of the NUbots Codebase.
+*
+* The NUbots Codebase is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* The NUbots Codebase is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+*
+* Copyright 2017 NUbots <nubots@nubots.net>
+*/
+
+syntax = "proto3";
+
+package message.output;
+
+// Buzzer message for the OpenCR subcontroller
+// This message takes an int32 as a value for  the buzzer frequency
+// Setting the frequency to 0 silences the buzzer
+
+message Buzzer {
+    int32 buzzer_frequency = 1;
```

Comment:
This is already a buzzer, no need to repeat the name of the message in the field
```suggestion
    int32 frequency = 1;
```